might have fixed the plotly issue?

miriammjr opened the pull request:
# Description

Uses line instead of marker for the hovering in charts. 

Fixes #587 

## Type of change

(Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

miriammjr committed:
might have fixed the plotly issue?

huss commented:
The checks failed due to linting issues. Hopefully it will pass after fixing them. These should be addressed before the PR will be reviewed.

miriammjr committed:
deleted comments to make the tests run

huss commented on src/client/app/containers/BarChartContainer.ts:
```diff
@@ -53,7 +53,11 @@ function mapStateToProps(state: State) {
 					hoverinfo: 'text',
 					type: 'bar',
-					marker: {color: getGraphColor(colorID, DataType.Meter)}
+					line: {
```
Since this is a bar graphic, I'm not certain we should be using the line attribute. Was there a specific reason to do this? Note the color of the bars differs from the same item on the line graphic. I don't think it used to do that.

Applies below as well.

huss replied:
fixed

huss reviewed (changes requested):
Does what was expected and seems to work fine. Just one comment on bar graphics to think about.

miriammjr committed:
reverted bar chart changes

huss committed:
resolve some issues

- make line color by line not marker call.
- group DataType fixed (was Meter).
- Decided to make line width 2 since I think it looks nicer.
- Fix bar so back to way it was so colors match line.
